bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -150,6 +150,19 @@ rules:
   - patch
   - update
   - watch
+{{- if .DNS_NAME_RESOLVER_ENABLE }}
+- apiGroups: ["network.openshift.io"]
+  resources:
+  - dnsnameresolvers
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
we'll need to split these rights between node and control pane once we finalize the ovn-k implementation. It is likely to be node can only read, and control-plane can create/update/delete

Reply:
Sure, for now, I've dropped create, delete, patch and update permissions from node. Let me know if it needs further refinement.